### PR TITLE
Many fix

### PR DESCRIPTION
**Pull request by first-leon**
Fixed the desynchronization of the alert name and alertmanager parameters.
Fixed an error that occurs in the case of an empty dataframe.
Change alert format.

### CODE REVIEW

File: src/bot/analytics.py [hunk 1]
```diff
@@ -61,13 +61,14 @@ def get_risks(df: pd.DataFrame, ratio_list: List[float]) -> pd.DataFrame:
...
 
 def get_distr(data) -> pd.DataFrame:
-    """This function calculates and returns a pivot table by risk levels"""
-
-    risk_distr = data.pivot_table(index="risk_rating", values=["collateral"], aggfunc=["sum"])
-    risk_distr.columns = ["ilk"]
-    risk_distr["percent"] = (risk_distr["ilk"] / risk_distr["ilk"].sum()) * 100
-
-    return risk_distr
+    if not data.empty:
```
Comment (madlabman) [inline src/bot/analytics.py:64]:
```suggestion
    if not data.empty:
        return pd.DataFrame()
```

File: prometheus/zones_changes.rule [hunk 2]
```diff
@@ -3,13 +3,26 @@ groups:
   - name: MakerBot
```
Comment (madlabman) [inline prometheus/zones_changes.rule:3]: I see NO tests for the updated rules.

File: src/bot/consts.py [hunk 3]
```diff
@@ -1,8 +1,17 @@
...
-HTTP_REQUESTS_DELAY = 3
+if os.getenv("HTTP_REQUESTS_RETRY"):
+    HTTP_REQUESTS_RETRY = int(os.getenv("HTTP_REQUESTS_RETRY"))
+else:
+    HTTP_REQUESTS_RETRY = 3
+
+if os.getenv("HTTP_REQUESTS_DELAY"):
+    HTTP_REQUESTS_DELAY = int(os.getenv("HTTP_REQUESTS_DELAY"))
+else:
+    HTTP_REQUESTS_DELAY = 3
```
Comment (madlabman) [inline src/bot/consts.py:17]: Move it to `src/bot/config.py`. There are already helpers for env variables.

File: src/bot/eth.py [hunk 4]
```diff
@@ -30,7 +30,7 @@
 VAT_ADDRESS = "0x35D1b3F3D7966A1DFe207aa4514C12a259A0492B"
 
 
-w3 = Web3(HTTPProvider(NODE_ENDPOINT))
+w3 = Web3(HTTPProvider(NODE_ENDPOINT, request_kwargs={'timeout': 90}))
```
Comment (madlabman) [inline src/bot/eth.py:33]: Run `black` please.

**Review by madlabman — CHANGES_REQUESTED** [review]

File: src/bot/config.py [hunk 5]
```diff
@@ -66,3 +66,13 @@ def getenv(name: str, astype: Type[T] = str, default: T = D, required: bool = Fa
     # WSS provider seems to be broken in python 3.10 and
     # doesn't work in the current flow. Magic asyncio fails happen.
     raise RuntimeError("Only http[s] Web3 provider endpoint supported")
+
+if os.getenv("HTTP_REQUESTS_RETRY"):
+    HTTP_REQUESTS_RETRY = int(os.getenv("HTTP_REQUESTS_RETRY"))
+else:
+    HTTP_REQUESTS_RETRY = 3
```
Comment (madlabman) [inline src/bot/config.py:73]:
```suggestion
HTTP_REQUESTS_RETRY = getenv("HTTP_REQUESTS_RETRY", int, default=3)
```

and the same for the one below

**Review by madlabman — APPROVED** [review]

**Review by PavelPoddubnyi — APPROVED** [review]

**Pull request merged**